src/auth/guards/magic-link.guard.ts

```diff
@@ -5,38 +5,28 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 import {
-  BadRequestException,
+  ExecutionContext,
   Injectable,
-  NotFoundException,
   UnauthorizedException,
 } from '@nestjs/common';
 import { AuthGuard } from '@nestjs/passport';
 
 @Injectable()
 export class MagicLinkGuard extends AuthGuard('magic-link') {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-explicit-any
-  handleRequest(err: any, user: any, info: any, context: any, status: any) {
+  handleRequest<User>(
+    err: Error,
+    user: User,
+    info: string,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    context: ExecutionContext,
```

Comment:
```suggestion
    _context: ExecutionContext,
```

You can do this to get rid of the warning